lib/bibliothecary/parsers/maven.rb

```diff
@@ -12,15 +12,15 @@ class Maven
       # e.g. "|    \\--- com.google.guava:guava:23.5-jre (*)"
       GRADLE_DEP_REGEX = /(\+---|\\---){1}/
 
-      # Project declaration lines so we know the current project name
-      # e.g. "Project ':submodules:test'" (this example is a project nested in submodules/test/ folder)
-      GRADLE_PROJECT_DECLARATION_REGEX = /Project '?:([^\s']+)'?/
```

Comment:
Since we don't interpolate the project name anymore, there's no need to track it.